Makefile

```diff
@@ -10,5 +15,18 @@ jar:
 release:
 	mvn -B release:prepare release:clean
 
-full:
-	mvn clean checkstyle:check cobertura:cobertura javadoc:jar package
+coverage:
+	mvn clean cobertura:cobertura
+
+doc:
+	mvn clean javadoc:javadoc
+
+see-coverage: coverage
+	cd target/site/cobertura; python -m SimpleHTTPServer
```

Comment:
Are these good to have in the makefile? Seems a little specific to our dev case (remote building). I don't mind keeping them in, just seems a little specific to our dev style.